<ui version="4.0" >
 <class>WeatherCfgBase</class>
 <widget class="QWidget" name="WeatherCfgBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabel1" >
       <property name="text" >
        <string>Location ID:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtID" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmbLocation" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch" >
       <property name="text" >
        <string>&amp;Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel1_4" >
     <property name="text" >
      <string>You can enter ID as part URL for your location on weather.com</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LinkLabel" native="1" name="lblLnk" />
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QComboBox" name="cmbUnits" >
       <item>
        <property name="text" >
         <string>Metric</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Standart</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" >
      <widget class="QSpinBox" name="edtDays" >
       <property name="maximum" >
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="TextLabel1_3" >
       <property name="text" >
        <string>Forecast days:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="TextLabel1_2" >
       <property name="text" >
        <string>Units:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LinkLabel</class>
   <extends>QWidget</extends>
   <header>simgui/linklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
